build.gradle

```diff
@@ -881,6 +888,11 @@ def buildTime() {
   return df.format(new Date())
 }
 
+// Get the date of the last git commit.
+def lastCommitDate() {
+  return grgit.log(maxCommits: 1).get(0).date
+}
```

Comment:
Probably need to check if grgit is available and fall back to using the current time if not.  There are always a few people who build from the downloaded source distribution rather than a git checkout.